coffee-chats/src/main/webapp/src/components/MainPage.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  Box, Button,
+  Container,
+  Grid,
+  Icon,
+  IconButton,
+  InputAdornment,
+  TextField,
+  Tooltip,
+  Typography
+} from "@material-ui/core";
```

Comment:
Stay consistent in import style, in fact styleguide says that imports are exempt to line-wrapping at 80-column limit, see: [here](https://google.github.io/styleguide/jsguide.html#es-module-imports)

Reply:
Yeah, that's what intellij does when you copy-paste between different files, I'll look into configuring it to maybe not do that